### PR TITLE
Install anishot from pypi

### PR DESCRIPTION
**Pull request by 4383**
## Overview
Package anishot with [openstack-pbr](https://docs.openstack.org/pbr/latest/) define anishot as a python package.
Introduce dependencies management via requirements.txt.
Move deprecated gflags to abls-py (https://pypi.python.org/pypi/absl-py)
You can now install anishot by using `pip install anishot`

## gflags to abls-py
gflags was deprecated and now it's part of the abls-py library, you need to migrate.

For how to migrate from gflags to abls-py more informations are availables [here](https://github.com/yilei/python-gflags/blob/1b1d6648690f682e8d6487f2a9ca949015f1a286/absl_migration/migration_guidelines.md)

## What's next?
If you accept these changes now you need to connect you travis-ci account with this project.
Pbr release project from git tags, if you want to autodeploy a new release on pypi from travis-ci job you need to create a git tag `git tag 0.1.0`  and to push it to your github repository.

You also need to configure a pypi account and to modify the `.travis.yml` file with your account informations (username, password), I recommand to you to encrypt your password with the travis CLI command.

More informations available on [travis documentation](https://docs.travis-ci.com/user/deployment/pypi/)

## Benefits
- manage more properly releases
- install anishot from pypi with pip
- install dependencies automaticaly
- remove deprecated dependency
- deploy automaticaly your releases

### CODE REVIEW

**Comment by redbitshift** [comment]
Wow thank you! I will review, and merge.

**Comment by 4383** [comment]
Hi @redbitshift!

To testing:
```shell
pip install pbr
python setup.py build
python setup.py install # or develop for develop purpose
```

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+anishot.egg-info/
+build/
```
Comment (sergey48k) [inline .gitignore:2]: Could you add __pycache__ here?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+absl-py
+imageio
```
Comment (sergey48k) [inline requirements.txt:2]: Could you add Pillow ?

**Review by sergey48k — CHANGES_REQUESTED** [review]
This looks great! Thank you for suggesting this. Could you please remove __pycache__ files from this PR?

File: .gitignore (hunk 1 above)
Comment (4383) [reply .gitignore:2]: Hi @sergey-surkov!

Sure, I've added some others usefull gitignore rules like, file swap from editors, tox unit tests temporary path (not used for the moment, but alrealdy ready for the futur).

File: requirements.txt (hunk 2 above)
Comment (4383) [reply requirements.txt:2]: Done!

**Comment by 4383** [comment]
Hi @sergey-surkov!

Requested changes applyed! I've already squashed my changes with the previous commit for keep all these changes together and make reverts easily if needed.

Have fun!

**Review by sergey48k — APPROVED** [review]
Looking good now!

**Pull request merged**